webapp/channels/src/components/admin_console/ldap_wizard/ldap_helpers.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import type {MessageDescriptor} from 'react-intl';
+
+import WithTooltip from 'components/with_tooltip';
+
+import type {GeneralSettingProps} from './ldap_wizard';
+
+import SchemaText from '../schema_text';
+
+/**
+ * LDAP-specific component for rendering help text with hover tooltip
+ */
+export const LDAPHelpTextWithHover: React.FC<{
+    baseText: string | JSX.Element | MessageDescriptor;
+    baseIsMarkdown?: boolean;
+    baseTextValues?: {[key: string]: any};
+    hoverText: string | JSX.Element | MessageDescriptor;
+    hoverIsMarkdown?: boolean;
+    hoverTextValues?: {[key: string]: any};
```

Comment:
I'd recommend removing `hoverIsMarkdown` and `hoverTextValues` and removing the option for `hoverText` to be a `JSX.Element` unless we really need any of those for the LDAP wizard. Those open up the possibility that the tooltip contains clickable items (most likely Markdown links), and I don't think `WithTooltip` properly supports that since you have to be careful to not hide the tooltip when the user goes to click on the link. Because of that, I think it'd be best to keep the new API simple

Reply:
Looking at #31286, I do see links in these hover tooltips which I'm almost certain won't work with `WithTooltip`. Either we need to change the design to remove the links from these tooltips or we're going to need a new component which behaves more like a modal so that users can click on the links inside of it